src/NUnitFramework/tests/Attributes/RangeAttributeTests.cs

```diff
@@ -64,7 +64,8 @@ public void MultipleAttributes()
         {
             Test test = TestBuilder.MakeParameterizedMethodSuite(typeof(RangeTestFixture), nameof(RangeTestFixture.MethodWithMultipleRanges));
 
-            Assert.That(from testCase in test.Tests select testCase.Arguments[0], Is.EqualTo(new[] { 1, 2, 3, 10, 11, 12 }));
+            var arguments = from testCase in test.Tests select testCase.Arguments[0];
+            Assert.That(arguments, Is.EqualTo(new[] { 1, 2, 3, 10, 11, 12 }).Or.EqualTo(new[] { 10, 11, 12, 1, 2, 3 }));
```

Comment:
Why not just Is.EquivalentTo? This seems to over specify the test.

Reply:
Thinking about it, I think you are right. I'll amend the PR.